packages/mobile/babel.config.js

```diff
@@ -1,7 +1,23 @@
 module.exports = (api) => {
   const babelEnv = api.env()
   const plugins = [
-    ['@babel/plugin-transform-react-jsx', { runtime: 'automatic' }]
+    ['@babel/plugin-transform-react-jsx', { runtime: 'automatic' }],
```

Comment:
wow amazing work making this work. is this based on an issue we could potentially reference? or maybe some comments? cause this is pretty wild

Reply:
Good call, let me add a comment. For reference here is the issue:
https://github.com/streamich/react-use/issues/483
https://github.com/streamich/react-use/blob/master/docs/Usage.md